package.json

```diff
@@ -102,13 +102,10 @@
     "babel-polyfill": "6.20.0",
     "babel-register": "6.18.0",
     "bluebird": "2.9.34",
-    "body-parser": "1.12.0",
```

Comment:
This appears to be used in APM, but not in the `package.json` there. We should probably add it there.

Reply:
haha, if we're looking at the same one I think it's a test case for a stack trace.  thoughts?

Reply:
Ah, yeah - it's not actually required.